src/components/Tooltip/index.js

```diff
@@ -0,0 +1,48 @@
+import { Typography, Button, Divider } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+const useStyles = makeStyles(({ typography }) => ({
+  tagline: {
+    margin: `${typography.pxToRem(40)} 0 ${typography.pxToRem(20)}`,
+  },
+  divider: {
+    height: typography.pxToRem(2),
+  },
+}));
+
+function Banner({ tagline, ctaText, ...props }) {
+  const classes = useStyles(props);
+  const handleClick = (event) => {
+    event.preventDefault();
+    // open toolbox
+  };
+  return (
+    <>
```

Comment:
Any reason we don't use fragment here?

Reply:
@kilemensi `<>` is a Fragment

Reply:
https://reactjs.org/docs/fragments.html#short-syntax @kilemensi

Reply:
Sorry @KhadijaMahanga , I meant any reason why **are we using** Fragment (as in instead of div/Box, etc.)

Reply:
@kilemensi aahh, okay. Let me add div or something, incase we need to manipulate the root